README.md

```diff
@@ -126,6 +126,17 @@ Run the command:
 $ make acc-test
 ```
 
+To run the acceptance test specific to a resource make use of the build-tags.
+
+`Running acceptance test without specifying any tags doesn't run any acceptance test at all.
```

Comment:
I see you changed  the acc-test target `acc-test: test-tags` so with that changing will this statement holds good?

Reply:
Made the change accordingly